doc/docker.md

```diff
@@ -110,11 +108,14 @@ Files/folders are mounted (bind) in the containers (nginx/certbot) defined in th
 
 The containers expect the following variables to be set in order to launch
 
+- ICUBAM_CONFIG_FILE: filename for the app's configuration file (e.g., `icubam.toml`)
 - SECRET_COOKIE
 - JWT_SECRET
 - GOOGLE_API_KEY
 - TW_KEY
 - TW_API
+- TW_API
+- DB_SALT
```

Comment:
Instead of repeating them here, could we link to the required env variables [here](https://github.com/icubam/icubam/blob/master/doc/install.md#configuration) (with `./install.md#configuration` relative link)?

Or at least have some way to list required env variables (excluding docker specific ones) in one place.

Reply:
> Is cloning the repo less convenient?

@rth  Here I was just trying to have a solution with the bare minimum (I was hoping for just one file, but then...). And working with branches is then ugly. Indeed, I can just clone and then clean the code after it's launched. Will do that.

Reply:
regarding the doc, I agree we should avoid duplication as much as possible to avoid inconsistencies. I'll update.

Reply:
replaced with relative link